pkg/controller/master-controller-manager/rbac/mapper.go

```diff
@@ -48,7 +48,7 @@ const (
 const (
 	saSecretsNamespaceName = "kubermatic"
 	// KubeOneNamespacePrefix is the kubeone namespace prefix.
-	KubeOneNamespacePrefix              = "kubeone"
+	KubeOneNamespacePrefix              = "kubeone-"
```

Comment:
I think such constants should be defined only in 1 place, and pkg/resources is convenience place for this (and it's already has it).

Reply:
Yes but resources import is not allowed here

Reply:
Done